### PR TITLE
Feat(apple): Cocoa direct install

### PR DESCRIPTION
**Pull request by brustolin**
## DESCRIBE YOUR PR
Explaining how to use cocoa pre-compiled frameworks

### CODE REVIEW

File: docs/platforms/apple/common/install/direct-install.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+To integrate Sentry into your Xcode project, follow these steps:
+
+1. Download the latest version of the SDK from the Sentry Cocoa [Releases page](https://github.com/getsentry/sentry-cocoa/releases).
+2. Each release contains the following four framework options:
+    - Sentry-Dynamic.xcframework.zip: Dynamic framework
+	- Sentry.xcframework.zip: Static framework
+	- SentrySwiftUI.xcframework.zip: Framework with SwiftUI support
+	- Sentry-WithoutUIKitOrAppKit.xcframework.zip: Framework without UIKit or AppKit linking and related features
```
Comment (philipphofmann) [inline docs/platforms/apple/common/install/direct-install.mdx:13]: `m`: Should we also mention that these are static for consistency?

File: docs/platforms/apple/common/install/direct-install.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Direct Install
```
Comment (philipphofmann) [inline docs/platforms/apple/common/install/direct-install.mdx:2]: `l`: Maybe we should call it manual?
```suggestion
title: Manual Install 
```

File: docs/platforms/apple/common/install/direct-install.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+To integrate Sentry into your Xcode project, follow these steps:
+
+1. Download the latest version of the SDK from the Sentry Cocoa [Releases page](https://github.com/getsentry/sentry-cocoa/releases).
+2. Each release contains the following four framework options:
+    - Sentry-Dynamic.xcframework.zip: Dynamic framework
+	- Sentry.xcframework.zip: Static framework
+	- SentrySwiftUI.xcframework.zip: Framework with SwiftUI support
+	- Sentry-WithoutUIKitOrAppKit.xcframework.zip: Framework without UIKit or AppKit linking and related features
+
```
Comment (philipphofmann) [inline docs/platforms/apple/common/install/direct-install.mdx:14]: `m`: I'm missing step 3 of actually adding this to the Xcode project. Especially, unexperienced developers might get confused.

**Review by philipphofmann — COMMENTED** [review]
Thanks for adding this. I have a few questions.

File: docs/platforms/apple/common/install/direct-install.mdx (hunk 1 above)
Comment (brustolin) [reply docs/platforms/apple/common/install/direct-install.mdx:13]:
```suggestion
	- SentrySwiftUI.xcframework.zip: Framework with SwiftUI support
	- Sentry-WithoutUIKitOrAppKit.xcframework.zip: Static framework without UIKit or AppKit linking and related features
```

File: docs/platforms/apple/common/install/direct-install.mdx (hunk 3 above)
Comment (brustolin) [reply docs/platforms/apple/common/install/direct-install.mdx:14]: Honestly I dont want to encourage unexperienced developers to use this approach. But we can at least use a generic step.

```suggestion
3. Import chosen framework into your Xcode project target.
```

File: docs/platforms/apple/common/install/direct-install.mdx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+To integrate Sentry into your Xcode project, follow these steps:
+
+1. Download the latest version of the SDK from the Sentry Cocoa [Releases page](https://github.com/getsentry/sentry-cocoa/releases).
+2. Each release contains the following four framework options:
+    - Sentry-Dynamic.xcframework.zip: Dynamic framework
+	- Sentry.xcframework.zip: Static framework
+	- SentrySwiftUI.xcframework.zip: Framework with SwiftUI support
+	- Sentry-WithoutUIKitOrAppKit.xcframework.zip: Static framework without UIKit or AppKit linking and related features
+3. Import chosen framework into your Xcode project target.
+### Usage Guidelines:
```
Comment (philipphofmann) [inline docs/platforms/apple/common/install/direct-install.mdx:15]:
```suggestion
3. Import chosen framework into your Xcode project target.

### Usage Guidelines:
```

File: docs/platforms/apple/common/install/direct-install.mdx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+title: Manual Install
```
Comment (philipphofmann) [inline docs/platforms/apple/common/install/direct-install.mdx:2]: `m`: We should also rename the file to `manual-install.mdx` to align.

File: docs/platforms/apple/common/install/direct-install.mdx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+description: "Integrate Sentry into your Xcode project by using our pre-compiled frameworks."
+---
+
+To integrate Sentry into your Xcode project, follow these steps:
+
+1. Download the latest version of the SDK from the Sentry Cocoa [Releases page](https://github.com/getsentry/sentry-cocoa/releases).
+2. Each release contains the following four framework options:
+    - Sentry-Dynamic.xcframework.zip: Dynamic framework
+	- Sentry.xcframework.zip: Static framework
+	- SentrySwiftUI.xcframework.zip: Framework with SwiftUI support
```
Comment (philipphofmann) [inline docs/platforms/apple/common/install/direct-install.mdx:12]: `l`: This is also static, I guess?
```suggestion
	- SentrySwiftUI.xcframework.zip: Static Framework with SwiftUI support
```

File: docs/platforms/apple/common/install/direct-install.mdx [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+	- Sentry-WithoutUIKitOrAppKit.xcframework.zip: Static framework without UIKit or AppKit linking and related features
+3. Import chosen framework into your Xcode project target.
```
Comment (philipphofmann) [inline docs/platforms/apple/common/install/direct-install.mdx:14]: `l`:
```suggestion
3. Import the chosen framework into your Xcode project target.
```

File: docs/platforms/apple/common/install/direct-install.mdx [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+### Usage Guidelines:
+
+	•	Use `Sentry-Dynamic`, `Sentry` (Static), or `Sentry-WithoutUIKitOrAppKit` independently. Only one of these should be included in your project at a time.
```
Comment (philipphofmann) [inline docs/platforms/apple/common/install/direct-install.mdx:17]: `l`: Or we also add static after `Sentry-WithoutUIKitOrAppKit` to be consistent.
```suggestion
	•	Use `Sentry-Dynamic`, `Sentry`, or `Sentry-WithoutUIKitOrAppKit` independently. Only one of these should be included in your project at a time.
```

**Review by philipphofmann — APPROVED** [review]
A few more minor things to fix, but LGTM.

**Review by philipphofmann — APPROVED** [review]
LGTM

**Pull request merged**